### ISSUE TITLE
Call arbitrary function on every element in list

### ISSUE THREAD

**Issue by sergeyksv** [timjansen/minified.js#52]
I know I can use each with callback function. But might be there is kind of shorthand?

Trivial example is how to call .play() method on EE("audio",{'@src':'something}). I know I can use [0].play(), but again might be there are shorthand?

**Comment by timjansen** [comment]
Not yet. There's call(), but it only calls a list of functions, and not the
methods stored in a property of the list members.

Will add a underscore-like invoke() function.

Thanks,
Tim

**Comment by sergeyksv** [comment]
@timjansen Thanks for your reply. Meanwhile my compliments for your hard work with this library, library itself and docs are awesome. I believe it is has kind of limited use areas just because size doesn't matter so much for majority of web apps. However for some use cases its hard to underestimate its benefits (balance between size and functionality). From that aspect I believe it is good that it has its own API, its not only small itself but allows to write smaller code around it.

**Comment by timjansen** [comment]
Yeah, I am very aware of the limitations because of the size. This is why
the next gen will be significantly larger (but still about 50% of jQuery if
you don't count the extra modules).

Tim

**Comment by sergeyksv** [comment]
I read about "minified reboot" that you announce. Honestly I believe there is a place for library how it is exist now, extremely small with precisely balanced functionality. So don't be surprised if community will choose 1.0 (current one) vs nextget :). However if you keep its core small enough keeping the rest in optional modules then who knows :). Honestly 50% of jQuery scares me :)
